lib/bundler/env.rb

```diff
@@ -4,89 +4,91 @@
 
 module Bundler
   class Env
-    def write(io)
-      io.write report
-    end
+    class << self
+      def write(io)
```

Comment:
Can you change these to `def self.` to avoid reopening the class and thus the indentation? That way we can avoid the massive diffs